python/servo/command_base.py

```diff
@@ -362,11 +362,6 @@ def build_env(self, gonk=False, hosts_file_path=None):
             env["OPENSSL_LIB_DIR"] = openssl_dir
             env['OPENSSL_INCLUDE_DIR'] = path.join(env["GONKDIR"], "external/openssl/include")
 
-        # FIXME: These are set because they are the variable names that
```

Comment:
Yeah, sorry I didn't see this PR come in! He does not need to change the code - just the comment. Sorry I wasn't clear!